packages/graphql-platform-core/src/graphql-platform/operation/mutation/update-one/value.ts

```diff
@@ -27,6 +27,14 @@ const updateInputHookProxyHandler: ProxyHandler<UpdateOneValue> = {
   ownKeys(update) {
     return [...update.resource.getComponentMap().keys()];
   },
+  getOwnPropertyDescriptor(update, propertyKey) {
+    if (!update.resource.getComponentMap().hasOwnProperty(propertyKey)) {
```

Comment:
I think it's `update.resource.getComponentMap().has(propertyKey)` or we'll have all the "Map"'s methods and we want only its elements

Reply:
typescript tells me that propertyKey is string|Symbol|number which does not fit has(string).
I think that in the destruct, only properties returned by ownKeys() + having an enumerable descriptor will be returned but I have to check.

Reply:
So it only returns the keys presents in ownKeys().
But I could also add a check in the `getOwnPropertyDescriptor` function to make sure that we don't proxy information from Map functions.
sth like ` if (!update.resource.getComponentMap().keys().includes(propertyKey)) return undefined`